### ISSUE TITLE
How to load a pretrained babkbone(swin-transformer) in the config file

### ISSUE THREAD

**Issue by rgbd-zml** [SwinTransformer/Swin-Transformer-Object-Detection#163]
I just want to load the pre-trained parameters of the backbone (swin-transformer). how to fix the config file? any help is thanks

**Comment by leesangjoon1** [comment]
I download the weights and put it in python file model settings like 
pretrained='address of weights'
under the type.

**Comment by chungdz** [comment]
@leesangjoon1 Could you kindly teach me how to start training and testing Cascade mask rcnn with pretrained swin-B as backbone on CoCo dataset? I kinda new to this field and don't know how to place the dataset and start training.

**Comment by suflaj** [comment]
In your config file, just add

```
load_from = "path/to/checkpoint"
```

So example, if you put your checkpoint in `checkpoints/some-checkpoint.pth`, you'd add

```
load_from = "checkpoints/some-checkpoint.pth"
```

to your config.
